src/main/java/com/vmware/vipclient/i18n/VIPCfg.java

```diff
@@ -68,44 +64,22 @@
     public static final String         CACHE_L3      = "CACHE_L3";
     public static final String         CACHE_L2      = "CACHE_L2";
 
-    public boolean isSubInstance() {
-        return isSubInstance;
-    }
-    
-    public void setSubInstance(boolean subInstance) {
-        isSubInstance = subInstance;
-    }
-
-    private boolean isSubInstance = false;
-
-    private VIPCfg() {
+    VIPCfg() {
```

Comment:
This is unnecessary.

Reply:
No. I don't think there are product using default constructor because VIPCfg is a singleton before.

Reply:
I know for a fact that there are products using the default constructor (e.g. Atlas).  They didn't want to use vipconfig.properties so they instantiated VIPCfg and set the properties using the setter methods with values from their application.yaml file.

Reply:
Makes sense. I removed it